guides/doc-Upgrading_and_Updating/topics/upgrading_process_overview.adoc

```diff
@@ -37,7 +41,8 @@ Upgrading {SmartProxy} takes approximately 10 - 30 minutes.
 
 * Ensure that you have sufficient storage space on your server. For more information, see {InstallingProjectDocURL}preparing-environment-for-satellite-installation[Preparing your Environment for Installation] in _Installing {ProjectServer} from a Connected Network_ and {InstallingSmartProxyDocURL}preparing-environment-for-capsule-installation[Preparing your Environment for Installation] in _Installing {SmartProxyServer}_.
 
-* Back up your {ProjectServer} and all {SmartProxyServer}s. For more information, see {AdministeringDocURL}backing-up-satellite-server-and-capsule-server[Backing Up {ProjectServer} and {SmartProxyServer}] in the _Administering {ProjectName} {ProductVersionPrevious}_ guide.
+* Back up your {ProjectServer} and all {SmartProxyServer}s. For more information, see {AdministeringDocURL}#backing-up-satellite-server-and-capsule-server
```

Comment:
```suggestion
* Back up your {ProjectServer} and all {SmartProxyServer}s. For more information, see {AdministeringDocURL}#backing-up-satellite-server-and-capsule-server[Backing Up {ProjectServer} and {SmartProxyServer}] in the _Administering {ProjectName} {ProductVersionPrevious}_ guide.
```
Without the text in [] this just prints a hyperlink. It's good practice to recommend the previous version of the guide because the backup and restore docs might shift from version to version.